packages/runtime/src/plugins/chrome-devtools.ts

```diff
@@ -0,0 +1,35 @@
+import { FederationRuntimePlugin } from '../index';
+import runtimeHelpers from '../helpers';
+
+declare global {
+  // eslint-disable-next-line no-var
+  var __INIT_VMOK_CHROME_DEVTOOL_PLUGIN__: boolean | undefined;
+}
+
+const chromeDevtoolsPlugin: () => FederationRuntimePlugin = function () {
+  return {
+    name: 'chrome-devtool',
+    beforeLoadRemoteSnapshot({ moduleInfo, options, ...info }) {
+      const { nativeGlobal } = runtimeHelpers.global;
+
+      if (options.inBrowser) {
+        const realLocalStorage = nativeGlobal.localStorage || localStorage;
+        const debugModuleInfoStr =
+          realLocalStorage.getItem('garfishModuleDebug');
```

Comment:
This key is the same as the internal key, and the plugin has been directly built in, which will cause data to be repeated by the plugin

Reply:
This plugin should be able to use global injection directly